subprojects/platform-jvm/src/test/groovy/org/gradle/jvm/toolchain/install/internal/AdoptOpenJdkRemoteBinaryTest.groovy

```diff
@@ -62,45 +61,17 @@ class AdoptOpenJdkRemoteBinaryTest extends Specification {
         13         | "Solaris"           | SystemInfo.Architecture.i386    | "https://api.adoptopenjdk.net/v3/binary/latest/13/ga/solaris/x32/jdk/hotspot/normal/adoptopenjdk"
     }
 
-    def "generates filename for jdk #jdkVersion on #operatingSystemName (#architecture)"() {
```

Comment:
Is there an equivalent test for the new naming logic?

About the naming logic, is it changed in a way that could cause conflict with JDKs downloaded by Gradle < 7.6?

Reply:
No, there is no equivalent test of this level (just some integration test coverage). I will add a test in a separate PR.
The names do change, but it shouldn't really break anything, because the name is not used for anything (as far as I know). They are only meant to avoid conflicting with each-other. What's important for auto-detection is the marker files placed in these folders and those don't change.